sdk/core/core/test/cmocka/test_az_span.c

```diff
@@ -759,6 +760,43 @@ static void az_span_copy_empty(void** state)
   assert_true(az_span_is_content_equal(az_span_copy(dst, AZ_SPAN_NULL), dst));
 }
 
+static void test_az_span_is_valid(void** state)
+{
+  (void)state;
+  assert_true(_az_span_is_valid({ 0 }, 0, true));
```

Comment:
nit: Consider adding similar test cases for `AZ_SPAN_NULL`.

Reply:
Done, in terms of comparing the contents with memcmp.